src/components/NcSelectUsers/NcSelectUsers.vue

```diff
@@ -303,7 +303,17 @@ defineProps<{
  * The `v-model` directive may be used for two-way data binding.
  */
 defineModel<IUserData>('modelValue')
+
+const emit = defineEmits<{
+	/**
+	 * Emitted when the user enters some query.
+	 * This can be used to asynchronously fetch more options.
+	 */
+	search: [string]
```

Comment:
Shall it be a model then?

Reply:
I also thought about that. Not sure if it is useful to pass the search as prop.
But would be fine with it.

Reply:
(also we cannot forward the search value to the VueSelect)